admin/src/main/java/com/exadel/frs/service/ModelService.java

```diff
@@ -96,6 +100,28 @@ public Model createModel(final ModelCreateDto modelCreateDto, final String orgGu
         return modelRepository.save(model);
     }
 
+    @Transactional
+    public Model cloneModel(
```

Comment:
What about faces? We also should copy them into new model. Please, check `MigrateController.java`  and `MigrationComponent.java`